packages/parser-typescript-config/src/schema.json

```diff
@@ -1,897 +1,922 @@
 {
-  "title": "JSON schema for the TypeScript compiler's configuration file",
```

Comment:
Was this file just auto-updated or did you also make manual edits? If the former, are those changes necessary for this PR?

Reply:
This was auto-updated :)